README.md

```diff
@@ -86,6 +86,16 @@ The extension provides a quick command to export your AsciiDoc file as DocBook.
 
 **ℹ️ Note:** Only DocBook 5 is supported.
 
+### Paste Image
+
+The extension provides a quick command to paste an image from clipboard.
+
+* Create a screenshot thats in the clipboard
+* Open the command palette - `ctrl+shift+p` or `F1` (Mac: `cmd+shift+p`)
+* Select _AsciiDoc: Paste Image_ (or use the shortcut `ctrl+alt+v`, Mac: `option+cmd+v`)
+* The png file with current timestamp is generated in the same folder
+* in the working document an image macro is inserted
```

Comment:
The last 2 bullets are not actions that the user must take, so I will probably convert them to a paragraph to describe what the user should see. What do you think?

Reply:
Hmmm good point, you're right we should probably be consistent. Since we are using passive sentence it should be:

An image file, named with a timestamp, is created, and an image macro is added to the document

Anyway, it really is a nitpick. Thanks for updating the documentation, I really appreciate it!